ACPs/77-reinventing-subnets/README.md

```diff
@@ -268,70 +275,71 @@ type RegisterSubnetValidatorTx struct {
 
-#### `SetSubnetValidatorWeightTx`
+#### `SetL1ValidatorWeightTx`
 
-`SetSubnetValidatorWeightTx` is used to modify the voting weight of a Subnet Validator. The specification of this transaction is:
+`SetL1ValidatorWeightTx` is used to modify the voting weight of a validator. The specification of this transaction is:
 
 ```golang
-type SetSubnetValidatorWeightTx struct {
+type SetL1ValidatorWeightTx struct {
     // Metadata, inputs and outputs
     BaseTx
-    // A SubnetValidatorWeightMessage payload
+    // An L1ValidatorWeightMessage payload
     Message warp.Message `json:"message"`
 }
 ```
 
 Applications of this transaction could include:
 
-- Increase the voting weight of a Subnet Validator if a delegation is made on the Subnet
-- Increase the voting weight of a Subnet Validator if the stake amount is increased (by staking rewards for example)
-- Decrease the voting weight of a misbehaving Subnet Validator
-- Remove an inactive Subnet Validator
+- Increase the voting weight of a validator if a delegation is made on the L1
+- Increase the voting weight of a validator if the stake amount is increased (by staking rewards for example)
+- Decrease the voting weight of a misbehaving validator
+- Remove an inactive validator
+
+The validation criteria for `L1ValidatorWeightMessage` is:
 
-The validation criteria for `SubnetValidatorWeightMessage` is:
 - `nonce >= minNonce`. Note that `nonce` is not required to be incremented by `1` with each successive validator weight update.
-- When `minNonce == MaxUint64`, `nonce` must be `MaxUint64` and `weight` must be `0`. This prevents Subnets from being unable to remove `nodeID` in a subsequent transaction.
-- If `weight == 0`, the Subnet Validator being removed must not be the last one in the set. If all Subnet Validators are removed, there are no valid Warp messages that can be produced to register new Subnet Validators through `RegisterSubnetValidatorMessage`. With no validators, block production will halt and the Subnet is unrecoverable. This validation criteria serves as a guardrail against this situation. A future ACP can remove this guardrail as users get more familiar with the new Subnet mechanics and tooling matures to fork a Subnet.
+- When `minNonce == MaxUint64`, `nonce` must be `MaxUint64` and `weight` must be `0`. This prevents L1s from being unable to remove `nodeID` in a subsequent transaction.
+- If `weight == 0`, the validator being removed must not be the last one in the set. If all validators are removed, there are no valid Warp messages that can be produced to register new validators through `RegisterL1ValidatorMessage`. With no validators, block production will halt and the L1 is unrecoverable. This validation criteria serves as a guardrail against this situation. A future ACP can remove this guardrail as users get more familiar with the new L1 mechanics and tooling matures to fork an L1.
```

Comment:
```suggestion
- If `weight == 0`, the validator being removed must not be the last one in the set. If all validators are removed, there are no valid Warp messages that can be produced to register new validators through `RegisterL1ValidatorMessage`. With no validators, block production will halt and the L1 is unrecoverable. This validation criteria serves as a guardrail against this situation. A future ACP can remove this guardrail as users get more familiar with the new L1 mechanics and tooling matures to fork a L1.
```

Reply:
> these indefinite articles are used [according to the sound](https://www.merriam-webster.com/grammar/is-it-a-or-an) that starts the following word, not the letter. So, we say “a dog” and “a balloon” or “an urgent message” and “an ant” but also “a useless pen” and “an hour.”

"L" said _out loud_ begins with a vowel sound; therefore the appropriate article is "an".
https://www.pristineword.com/a-an/#tit05